config/CLIConfiguration.ts

```diff
@@ -405,25 +408,23 @@ class CLIConfiguration {
     safelyApplyUpdates('parentAccountId', parentAccountId);
 
     const completedAccountConfig = nextAccountConfig as FlatAccountFields_NEW;
-
+    if (!Object.hasOwn(this.config, 'accounts')) this.config.accounts = [];
     if (currentAccountConfig) {
       logger.debug(
         i18n(`${i18nKey}.updateAccount.updating`, {
           accountId,
         })
       );
       const index = this.getConfigAccountIndex(accountId);
-      this.config.accounts[index] = completedAccountConfig;
+      if (index < 0) this.config.accounts.push(completedAccountConfig);
```

Comment:
If this util is called `updateAccount()`, what's the scenario where we'd call it without the account already existing in the config?

I poked around in this file and it looks like there's no `addAccount()` util. Maybe we should name this one `addOrUpdateAccount()` for clarity? This is just a nit.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/0559c584504dc458721cef1dced436eaf0e88862